frontend/projects/upgrade/src/app/features/dashboard/home/components/modal/experiment-status/experiment-status.component.html

```diff
@@ -13,9 +13,9 @@
         disabled="true"
         [(value)]="experimentInfo.state"
       >
-        <mat-option class="ft-14-400" *ngFor="let status of experimentStatus" [value]="status.value">
-          {{ status.value | experimentState }}
-        </mat-option>
+      <mat-option class="ft-14-400" *ngFor="let status of experimentStatus" [value]="experimentInfo.state">
```

Comment:
I think we can remove the `*ngFor="let status of experimentStatus"` because we only need to display one option.